src/test/java/io/vavr/collection/euler/Utils.java

```diff
@@ -115,4 +115,15 @@ static boolean isPalindrome(String val) {
     static boolean isPalindrome(int val) {
         return isPalindrome(Long.toString(val));
     }
+
+    static Stream<Long> pentagonal() {
+        return Stream.of(Tuple.of(1L, 1)).appendSelf(self ->
+                self.map(t ->
+                        Tuple.of((t._2 + 1) * (3L * (t._2 + 1) - 1) / 2, t._2 + 1)))
+                .map(t -> t._1);
```

Comment:
@cal101 could you provide the implementation you suggested? That way @smthelusive can verify if the test is still green...